realm/realm-library/src/objectServer/java/io/realm/mongodb/sync/SyncConfiguration.java

```diff
@@ -793,6 +799,17 @@ public Builder rxFactory(RxObservableFactory factory) {
             return this;
         }
 
+        /**
+         * Sets the {@link FlowFactory} used to create coroutines Flows from Realm objects.
+         * The default factory is {@link RealmFlowFactory}.
+         *
+         * @param factory factory to use.
+         */
+        public Builder flowFactory(FlowFactory factory) {
+            flowFactory = factory;
```

Comment:
We should guard against null here.

Reply:
See https://github.com/realm/realm-java/pull/7191#discussion_r522848928

Reply:
Got it - https://github.com/realm/realm-java/pull/7191#discussion_r522857800